test/arithmetic_test.rb

```diff
@@ -8,6 +8,36 @@ class Measured::ArithmeticTest < ActiveSupport::TestCase
     @four = Magic.new(4, :magic_missile)
   end
 
+  test 'can check for finite?' do
+    assert Magic.new(0, :magic_missile).finite?
+    refute Magic.new(Float::INFINITY, :magic_missile).finite?
```

Comment:
I love infinity magic missiles. I really do.

Reply:
Who wouldn't!

We actually have legitimate use cases where use use infinite? measures to represent unlimited capacities.